ee/packages/presence/src/Presence.ts

```diff
@@ -39,9 +39,17 @@ export class Presence extends ServiceClass implements IPresence {
 			}
 		});
 
-		this.onEvent('license.module', ({ module, valid }) => {
+		this.onEvent('license.module', async ({ module, valid }) => {
 			if (module === 'scalability') {
 				this.hasLicense = valid;
+
+				if (!this.broadcastEnabled && valid) {
+					// broadcast should always be enabled if license is active (unless the troubleshoot setting is on)
+					const presenceBroadcastDisabled = await Settings.findOneById('Troubleshoot_Disable_Presence_Broadcast');
```

Comment:
checking this setting is not wrong, but it is not a concern of this package..

we need to create an alternate mechanism to properly respect this setting, as it is already being "ignored" in other places.